.env-template

```diff
@@ -1,5 +1,6 @@
 # REQUIRED VARIABLES
 NEXT_PUBLIC_API_HOSTNAME=http://backend:8000
+NEXT_PUBLIC_HAS_CUSTOM_LOGO=false
```

Comment:
i would probably move this out to its own header, maybe `# CORAL WEB` since it's not a required variable